src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -214,17 +205,220 @@ The `EvmNode` object includes the following items for a given coin or token:
+
+| Parameter                   | Type    | Description                                                                                                     |
+|-----------------------------|---------|-----------------------------------------------------------------------------------------------------------------|
+| channel_id                  | string  | Optional. Unique string identifying a channel by its ID.                                                        |
+| counterparty_node_id        | string  | Optional. A hexidecimal string identifying a counterparty node. |
+| funding_tx                  | string  | Optional. A transaction ID which added funds.                   |
+| from_funding_value_sats     | integer | Optional. The minimum value of channel funding in satoshis.     |
+| to_funding_value_sats       | integer | Optional. The maximum value of channel funding in satoshis.     |
+| is_outbound                 | boolean | Optional. If `true`, limits the response to outbound channels only. |
+| from_balance_msat           | integer | Optional. The minimum channel balance in millisatoshis.     |
+| to_balance_msat             | integer | Optional. The maximum channel balance in millisatoshis.     |
+| from_outbound_capacity_msat | integer | Optional. The minimum outbound capacity of the channel balance in millisatoshis.  |
+| to_outbound_capacity_msat   | integer | Optional. The maximum outbound capacity of the channel balance in millisatoshis.  |
+| from_inbound_capacity_msat  | integer | Optional. The minimum inbound capacity of the channel balance in millisatoshis.   |
+| to_inbound_capacity_msat    | integer | Optional. The maximum inbound capacity of the channel balance in millisatoshis.   |
+| confirmed                   | boolean | Optional. If `true`, channel is confirmed NEED MORE INFO |
+| is_usable                   | boolean | Optional. If `true`, channel is usable NEED MORE INFO    |
+| is_public                   | boolean | Optional. If `true`, channel is public NEED MORE INFO    |
+
+
+### LightningPayment
+
+| Parameter            | Type    | Description                                                                                                                           |
+|----------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------|
+| type                 | string  | The payment type. Accepted values are `invoice` or [`keysend`](https://lightning.readthedocs.io/lightning-keysend.7.html).                                                                         |
+| invoice              | string  | Only used if `type` is `invoice`. An identifying string which represents the invoice.                                                 |
+| destination          | string  | Only used if `type` is `keysend`. A pubkey or address which will receive the payment.                                                 |
```

Comment:
destination is the `node_pubkey` which is also the node address in lightning context, not to be confused with an onchain address.

Reply:
thanks, updated.